.eslintignore

```diff
@@ -48,3 +49,8 @@ packages/graphiql/lsp
 
 # Vendored files
 /packages/graphiql/test/vendor
+
+
+# Autogenerated CHANGELOGs
```

Comment:
lets just make it all MDs, as we also have the ones at the base?

Reply:
We'll have to make it run pretty after creating these then, but I'm happy with that.

Reply:
this is a new addition in package.json:

```json
{
 "lint-staged": {
    "*.{js,ts,jsx,tsx}": [
      "eslint --fix",
      "prettier --write",
      "git add"
    ],
    "*.{md}": [
      "prettier --write",
      "git add"
    ]
  },
}
```

Reply:
we also perform a prettier check in CI